astro/src/components/docs/release-notes/GitHubIssue.astro

```diff
@@ -0,0 +1,31 @@
+---
+export interface Props {
```

Comment:
I think the actual correct Props definition would be:

`export type Props = { issue: string } | { resolvedIn: string, viaIssue: string };`

Which may feel a little nitpicky but I bring it up because it tells me that this component might be trying to be two things at once, and maybe it should be two components instead.

Reply:
Thanks for the feedback! I went back and forth a bit.. creating one or two components. There are probably two uses cases in the release notes for this - both accounted for in this one component. So creating two wouldn't be so bad. 

I am totally fine with creating two if that is the better pattern.